src/Tribe/JSON_LD/Event.php

```diff
@@ -32,6 +32,21 @@ public static function instance( $name = null ) {
 	}
 
 	/**
+	 * Converts the start and end dates to the local timezone
+	 *
+	 * @param  string $event_tz_string
```

Comment:
Param tags are in the wrong order - should match the method signature (ie `@param string $date` ought to be listed first)